lib/new_relic/cli/commands/deployments.rb

```diff
@@ -163,24 +163,24 @@ def set_params_v2(request)
   def options
     OptionParser.new(%Q(Usage: #{$0} #{self.class.command} [OPTIONS] ["description"] ), 40) do |o|
       o.separator("OPTIONS:")
-      o.on("-a", "--appname=NAME", String,
-        "Set the application name.",
-        "Default is app_name setting in newrelic.yml. Available only when using API v1.") { |e| @appname = e }
-      o.on("-i", "--appid=ID", String,
-        "Set the application ID",
-        "If not provided, will connect to the New Relic collector to get it") { |i| @application_id = i }
-      o.on("-e", "--environment=name", String,
-        "Override the (RAILS|RUBY|RACK)_ENV setting",
+      o.on('-a', '--appname=NAME', String,
+        'Set the application name.',
+        'Default is app_name setting in newrelic.yml. Available only when using API v1.') { |e| @appname = e }
+      o.on('-i', '--appid=ID', String,
+        'Set the application ID',
+        'If not provided, will connect to the New Relic collector to get it') { |i| @application_id = i }
+      o.on('-e', '--environment=name', String,
+        'Override the (RAILS|RUBY|RACK)_ENV setting',
         "currently: #{control.env}") { |e| @environment = e }
-      o.on("-u", "--user=USER", String,
-        "Specify the user deploying, for information only",
+      o.on('-u', '--user=USER', String,
+        'Specify the user deploying, for information only',
         "Default: #{@user || '<none>'}") { |u| @user = u }
-      o.on("-r", "--revision=REV", String,
-        "Specify the revision being deployed. Required when using New Relic REST API v2") { |r| @revision = r }
-      o.on("-l", "--license-key=KEY", String,
-        "Specify the license key of the account for the app being deployed") { |l| @license_key = l }
-      o.on("-c", "--changes",
-        "Read in a change log from the standard input") { @changelog = STDIN.read }
+      o.on('-r', '--revision=REV', String,
+        'Specify the revision being deployed. Required when using New Relic REST API v2') { |r| @revision = r }
+      o.on('-l', '--license-key=KEY', String,
+        'Specify the license key of the account for the app being deployed') { |l| @license_key = l }
+      o.on('-c', '--changes',
+        'Read in a change log from the standard input') { @changelog = STDIN.read }
```

Comment:
I love seeing these single quotes. 

Isn't there a built-in linter for changing double quotes to singles for areas without interpolation? I wonder if that might be worth turning on?

Reply:
You can tell RuboCop to insist on the use of single quotes for strings without interpolation or you can tell RuboCop to insist on double quotes everywhere. I believe RuboCop currently defaults to the first one of those, but there has been well reasoned arguments for the second approach as well.

I believe we are currently disabling that cop and allowing any kind of quotes at any time regardless of interpolation.

If the devs decide on a standard, we can update the RuboCop config accordingly. Until then, I've been operating with my personal preference of RuboCop defaults.

Reply:
For the record! I prefer single quotes. Double quotes alert me to interpolation, so I'd love to see this single quote cop enabled.

Reply:
My preference is also single quotes unless in interpolation, though it can get hairy if we have apostrophes in our strings.